### PR TITLE
Documentation of the simulation configuration

### LINKED ISSUE #211: Provide documentation for grid referene systems
Currently, no documentation is provided for configuring reference systems in `simona.gridConfig.refSystems` in a way that fits some given grid.

### LINKED ISSUE #334: Documentation of basic configuration (User's Guide) and advanced configuration (Developer's Guide)
In the User's Guide, the documentation of the simulation inputs must be completed (use of default and custom configuration). In the Developer's Guide the advanced configuration must be completed (grid configuration, power flow configuration, etc.). This includes #211.

### PR DESCRIPTION
**Pull request by vickybung1**
Resolves #334 
Resolves #211

### CODE REVIEW

**Review by danielfeismann — CHANGES_REQUESTED** [review]
Thanks for your work done here. This already looks good to me and I will be happy to have this in fast. I have some (minor) things where I think it will further improve the documentation. Please have a look and let me know if you see things different.

File: docs/readthedocs/models/reference_system.md [hunk 1]
```diff
@@ -17,14 +17,11 @@ The reference system is built up by specifying the included voltage levels. The
...
-`{sNom = "60 MVA", vNom = "20 kV", voltLvls = [{id = "MS", vNom = "20 kV"}]},`
-
-`{sNom = "600 MVA", vNom = "110 kV", voltLvls = [{id = "HS", vNom = "110 kV"}]},`
-
-`{sNom = "1000 MVA", vNom = "380 kV", voltLvls = [{id = "HoeS", vNom = "380 kV"}]}`
-
-`]`
+```
+simona.gridConfig.refSystems = [
+    {sNom = "100 kVA", vNom = "0.4 kV", voltLvls = [{id = "NS", vNom = "0.4 kV"}]},
```
Comment (vickybung1) [inline docs/readthedocs/models/reference_system.md:22]: Should the identifier of the voltage levels be adapted?

Comment (danielfeismann) [reply docs/readthedocs/models/reference_system.md:22]: ATM we leave it as it is. I'll take this into our group and if there is a vote for changing it we will create a new issue.

**Review by danielfeismann — APPROVED** [review]
This looks good to me! Thanks a lot for your work done here.

**Pull request merged**